src/containers/KanbanBoard/KanbanBoard.jsx

```diff
@@ -6,14 +6,26 @@ import AddCard from '../AddCard';
 import Header from '../../components/Header';
 import CardDetail from '../CardDetail';
 import EditCard from '../EditCard';
+import Login from '../Login';
+import Register from '../Register';
 
 class KanbanBoard extends Component {
   state = {
     addFormOpen: false,
     detailOpen: false,
-    editFormOpen: false
+    editFormOpen: false,
+    loginOpen: false,
+    registerOpen: false
   };
 
+  shouldBlur = () => {
+    const { addFormOpen, detailOpen, editFormOpen, loginOpen, registerOpen } = this.state
+    if (addFormOpen || detailOpen || editFormOpen || loginOpen || registerOpen) {
```

Comment:
Could consolidate this into: `return (addFormOpen || detailOpen || editFormOpen || loginOpen || registerOpen)`